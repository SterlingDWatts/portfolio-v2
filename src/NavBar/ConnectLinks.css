.ConnectLinks {
    display: flex;
    flex-direction: row;
    justify-content: space-evenly;
}

@media screen and (min-width: 900px) {
    .ConnectLinks {
        max-width: 944px;
        justify-content: space-between;
        margin: 0 auto;
    }
}
