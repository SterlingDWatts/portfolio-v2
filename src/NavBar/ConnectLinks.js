import React, { Component } from 'react';
import './ConnectLinks.css';

class ConnectLinks extends Component {

    render() {
        return (
            <ul className='ConnectLinks'>
                <li><a href="https://github.com/SterlingDWatts" target="_blank"><i className="fab fa-github"></i></a></li>
                <li><a href="https://www.linkedin.com/in/sterlingdwatts/" target="_blank"><i className="fab fa-linkedin-in"></i></a></li>
                <li><a href="https://www.facebook.com/SterlingDWatts" target="_blank"><i className="fab fa-facebook-f"></i></a></li>
                <li><a href="https://www.instagram.com/sterlingdwatts" target="_blank"><i className="fab fa-instagram"></i></a></li>
                <li><a href="https://twitter.com/SterlingDWatts" target="_blank"><i className="fab fa-twitter"></i></a></li>
            </ul>
        );
    }

}

export default ConnectLinks;
