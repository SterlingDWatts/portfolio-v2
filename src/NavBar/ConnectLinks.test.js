import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ConnectLinks from './ConnectLinks';

describe(`ConnectLinks Component`, () => {

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<ConnectLinks />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
            .create(<ConnectLinks />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

})
