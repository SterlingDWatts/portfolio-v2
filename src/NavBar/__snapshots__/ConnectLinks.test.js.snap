// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectLinks Component renders the UI as expected 1`] = `
<ul
  className="ConnectLinks"
>
  <li>
    <a
      href="https://github.com/SterlingDWatts"
      target="_blank"
    >
      <i
        className="fab fa-github"
      />
    </a>
  </li>
  <li>
    <a
      href="https://www.linkedin.com/in/sterlingdwatts/"
      target="_blank"
    >
      <i
        className="fab fa-linkedin-in"
      />
    </a>
  </li>
  <li>
    <a
      href="https://www.facebook.com/SterlingDWatts"
      target="_blank"
    >
      <i
        className="fab fa-facebook-f"
      />
    </a>
  </li>
  <li>
    <a
      href="https://www.instagram.com/sterlingdwatts"
      target="_blank"
    >
      <i
        className="fab fa-instagram"
      />
    </a>
  </li>
  <li>
    <a
      href="https://twitter.com/SterlingDWatts"
      target="_blank"
    >
      <i
        className="fab fa-twitter"
      />
    </a>
  </li>
</ul>
`;
